### PR TITLE
Zy/477 cluster implement ewkm

### LINKED ISSUE #477: CLUSTER: Implement EWKM

### PR DESCRIPTION
**Pull request by zheyxu**
# Pull Request Details

## What issue does this PR address

- CLUSTER: Implement EWKM

- Link to associated issue: #477

## Checklist

Complete the check-list below to ensure your branch is ready for PR.

Flutter Style Guide: https://survivor.togaware.com/gnulinux/flutter-style.html

- [x] Screenshots included in linked issue
- [x] Changes adhere to the style and coding guideline
- [x] No confidential information
- [x] No duplicated content
- [x] No lint check errors related to your changes (`make prep` or `flutter analyze lib`)
- [x] Pre-exisiting lint errors noted: [HERE]
- [x] Integration test `make qtest.tmp` screenshot included in issue
- [x] Tested on device:
  - [ ] Linux
  - [x] MacOS
  - [ ] Windows
- [x] Added two reviewers

## Finalising

Once PR discussion is complete and reviewers have approved:

- [ ] Merge dev into the branch
- [ ] Resolve any conflicts
- [ ] Add one line summary into CHANGELOG.md
- [ ] Bump appropriate version number in pubspec.yaml
- [ ] Push to git repository and review
- [ ] Merge PR into dev (gjwgit)

### CODE REVIEW

**Comment by gjwgit** [comment]
Looks good - I will test run and review code.

File: assets/r/model_build_cluster_kmeans.R [hunk 1]
```diff
@@ -43,6 +43,7 @@ set.seed(CLUSTER_SEED)
 # The 'reshape' package provides the 'rescaler' function.
 
 library(reshape)
+library(wskm, quietly=TRUE)  # Load the wskm package for EWKM
```
Comment (gjwgit) [inline assets/r/model_build_cluster_kmeans.R]: Is wskm required for kmeans?

**Review by gjwgit — CHANGES_REQUESTED** [review]
Please review my updates. Thanks.

**Review by zheyxu — COMMENTED** [review]
Checked.

**Pull request merged**